sensu/plugins/check-cinder-sessions.py

```diff
@@ -76,11 +76,11 @@ def main():
         expected_vols = _get_local_active_instance_volumes()
         connected_vols = _get_active_iscsi_volumes()
 
-        for vol in expected_vols:
-            if not vol['id'] in connected_vols:
-                print "Volume %s is 'in-use' but not " \
-                      "connected" % vol['id']
-                failure = True
+        for vm_and_vols in expected_vols:
+            if not vm_and_vols[1][0]['id'] in connected_vols:
+                print "The instance %s has the volume %s attached, " \
```

Comment:
I know you're just modifying this, but can you use the print() function rather than the print statement? The function is future compatible, the statement will break in the future.